dnf/crypto.py

```diff
@@ -139,6 +139,15 @@ def log_key_import(keyinfo):
     logger.critical("%s", msg)
 
 
+def log_dns_key_import(keyinfo, dns_result):
+    log_key_import(keyinfo)
+    if dns_result == dnf.dnssec.Validity.VALID:
+        logger.critical(_('Verified using DNS record with DNSSEC signature.'))
+    elif dns_result == dnf.dnssec.Validity.PROVEN_NONEXISTENCE:
+        logger.critical(_('Verified using DNS record. But NOT signed using DNSSEC.'))
```

Comment:
Just remove the elif: please. Unsigned key is not proof for anything, just say not verified in else.

Reply:
I agree after reading the explanation. Thank you.